runtimes/server-interface/workspace.ts

```diff
@@ -27,5 +29,16 @@ export type Workspace = {
         readFile: (path: string) => Promise<string>
         isFile: (path: string) => Promise<boolean>
         remove: (dir: string) => Promise<void>
+        writeFile: (
+            file: FileHandle | string | Buffer | URL,
```

Comment:
Should we expose `FileHandle` to the server? It mean cross-platform server would need to depend on type from NodeJs-specific type.

Reply:
Good catch! I've removed the types that we might not need for the notification server implementation. We can easily extend it later if needed.